src/app/side-bar/dashboard/dashboard.component.html

```diff
@@ -1 +1,40 @@
-<p>dashboard works!</p>
+<div  fxLayout="column" class="dashboard-container">
+  <div fxLayoutAlign="center center" class="width--full" >
+    <p>No. of contacts</p>
+    <div fxFlex="70" class="dashboard-container__graph">
+      <p-chart type="line" [data]="lineStylesData" height="300" [options]="options"></p-chart>
+    </div>
+  </div>
+  <p fxLayoutAlign="center center">Days [2022]</p>
+  <div fxLayout="column" class="card-container">
+    <div fxlayout="row" fxLayoutAlign="space-between center" class="card-title">
+      <h5>How to use the portal</h5>
+      <span  fxLayout="row" fxLayoutAlign="start center" fxLayoutGap="10px">
```

Comment:
little bit off with the alignment, please center it vertically.

Reply:
did you mean in design???

Reply:
resolve ...................Thank you